<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1891</width>
    <height>955</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="canvas">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="add_object_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать объект</string>
    </property>
   </widget>
   <widget class="QComboBox" name="objects_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_object_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_light_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="add_light_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="add_light_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать источник света</string>
    </property>
   </widget>
   <widget class="QLabel" name="ambient_light_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Окружающий свет</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ambient_spin">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>370</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="render_button">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рендер</string>
    </property>
   </widget>
   <widget class="QLabel" name="objects_in_scene_label">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Объекты в сцене</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_object_button">
    <property name="geometry">
     <rect>
      <x>1610</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QListView" name="scene_list">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>80</y>
      <width>321</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="params_label">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>282</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Параметры</string>
    </property>
   </widget>
   <widget class="QLabel" name="offset_label">
    <property name="geometry">
     <rect>
      <x>1454</x>
      <y>325</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Смещение</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotate_label">
    <property name="geometry">
     <rect>
      <x>1584</x>
      <y>326</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
   </widget>
   <widget class="QLabel" name="scale_label">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>327</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="offset_x_spin">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>365</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="x_label">
    <property name="geometry">
     <rect>
      <x>1403</x>
      <y>365</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_x_spin">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>365</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_x_spin">
    <property name="geometry">
     <rect>
      <x>1698</x>
      <y>365</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.200000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_y_spin">
    <property name="geometry">
     <rect>
      <x>1698</x>
      <y>406</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.200000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="offset_y_spin">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>406</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_y_spin">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>406</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="y_label">
    <property name="geometry">
     <rect>
      <x>1403</x>
      <y>406</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="z_label">
    <property name="geometry">
     <rect>
      <x>1403</x>
      <y>445</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_z_spin">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>445</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_z_spin">
    <property name="geometry">
     <rect>
      <x>1698</x>
      <y>445</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.200000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="offset_z_spin">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>445</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="texture_flag">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>500</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Текстура</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_texture_button">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>530</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="texture_img">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>510</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="color_flag">
    <property name="geometry">
     <rect>
      <x>1411</x>
      <y>640</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="color_add_button">
    <property name="geometry">
     <rect>
      <x>1411</x>
      <y>670</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="reflection_label">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>720</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отражение</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="reflection_spin">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>750</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="transparency_spin">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>820</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="transparency_label">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>790</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Прозрачность</string>
    </property>
   </widget>
   <widget class="QLabel" name="glitter_label">
    <property name="geometry">
     <rect>
      <x>1573</x>
      <y>721</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Блеск</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="glitter_spin">
    <property name="geometry">
     <rect>
      <x>1573</x>
      <y>751</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QGraphicsView" name="color_preview">
    <property name="geometry">
     <rect>
      <x>1551</x>
      <y>669</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="intensity_spin">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>530</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="intensity_label">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>497</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Интенсивность</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1891</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
